tools/clang/unittests/HLSL/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ target_link_libraries(clang-hlsl-tests PRIVATE
   ${TAEF_LIBRARIES}
   ${DIASDK_LIBRARIES}
   ${D3D12_LIBRARIES}
+  shlwapi
```

Comment:
What was the dependency that required this to be added?  I think we might have to rely on a different lib or even API, since this one might be deprecated.